### PR TITLE
Added CSV import examples + doc

### PR DESCRIPTION
**Pull request by gabrielfs7**
Added CSV import examples + doc

https://oat-sa.atlassian.net/browse/ADF-38

Instructions are here: https://github.com/oat-sa/extension-tao-itemqti/blob/feature/ADF-38%2Fcsv-tabular-import-template/model/import/README.md

Notes:

- The XML template is not required and we can use the `taoQtiItem` classes to build the item based on the csv received.
- All the cases for the BOSA MVP are described in the README.md

### CODE REVIEW

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: templates/import/import.sample.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
+QCM2000,What is the <strong>largest</strong> mountain?                    ,1      ,0          ,1          ,en-US   ,Mount Olympus,0             ,Mount Kilimanjaro,2             ,Mount Everest,5             ,Pichu Pichu,3             ,        ,              ,choice_3           ,GEO_b                ,15                 ,
+QCM3000,Which is the <strong>smallest</strong> planet of our solar system?,       ,           ,           ,        ,Earth        ,4             ,Uranus           ,2             ,Jupiter      ,1             ,Neptune    ,3             ,Mercury ,5             ,"choice_5"         ,                     ,35                 ,
+QCM4000,Which of those are cities of Italy?                               ,       ,           ,           ,        ,Venice       ,1             ,Rome             ,1             ,New York     ,0             ,Paris      ,0             ,        ,0             ,"choice_1,choice_2","""geography"""      ,20                 ,"1,2"
```
Comment (shpran) [inline templates/import/import.sample.csv:5]: If you remove empty spaced after quotes - github will prettify your csv 😄 
image

```suggestion
QCM3000,Which is the <strong>smallest</strong> planet of our solar system?,       ,           ,           ,        ,Earth        ,4             ,Uranus           ,2             ,Jupiter      ,1             ,Neptune    ,3             ,Mercury ,5             ,"choice_5",                     ,35                 ,
QCM4000,Which of those are cities of Italy?                               ,       ,           ,           ,        ,Venice       ,1             ,Rome             ,1             ,New York     ,0             ,Paris      ,0             ,        ,0             ,"choice_1,choice_2","""geography""",20                 ,"1,2"
```

**Review by shpran — APPROVED** [review]
Good job! 👍🏻

File: templates/import/import.sample.csv (hunk 1 above)
Comment (gabrielfs7) [reply templates/import/import.sample.csv:5]: Indeed, but it is a dilemma, cause on the other hand it become hard to read on the text editor :/ 

So I preferred to make it easier to read in the text editor. Is it ok for you?

**Pull request merged**